README.md

```diff
@@ -150,3 +162,32 @@ curl http://192.168.0.42:8123/client/foo/http/127.0.0.1:8125/bigfile
 where `foo` is the hardcoded client ID for this test server.
 
 This test server only supports GET requests.
+
+### HA Usage
+
+To test remotedialer in HA mode, first start the dummy server from an appropriate directory, eg.:
+
+```shell
+cd /tmp
+mkdir www
+cd www
+echo 'hello' > bigfile
+/path/to/dummy -listen :8125
+```
+
+Then start two peer remotedialer servers with the `-peers id:token:url` flag:
+
+```shell
+./server/server -debug -id first -token aaa -listen :8123 -peers second:aaa:ws://localhost:8124/connect &
+./server/server -debug -id second -token aaa -listen :8124 -peers first:aaa:ws://localhost:8123/connect
+```
+
+Then connect a client to the first server, eg:
+```shell
+./client/client -connect ws://localhost:8123/connect
```

Comment:
Could you also add the `-id foo` argument explicitely in this example to make it clear that the `foo` in the curl example refers to the dummy server?

Reply:
addressed in b8c6ac7e7aa74a8161c70e34e982f9e576624e1d